README.md

```diff
@@ -61,6 +61,41 @@ optimizt --webp path/to/directory
 optimizt `find . -type f -name '*.jpg'`
 ```
 
+## About “lossy” and “lossless”
+
+### JPEG
+
+In lossy mode [jpegoptim](https://github.com/tjko/jpegoptim) is used with flags: `--strip-all`,
+`--all-progressive`, `--max=80`.
+
+Lossless mode uses a [Guetzli](https://github.com/google/guetzli) encoder with `--quality 90` flag.
+
+Guetzli is aims for excellent compression density at high visual quality.
+
+If you re-optimize the same file in lossless mode, the file size may decrease, but the visual quality will also degrade.
+
+### PNG
+
+For both modes, [pngquant](https://github.com/kornelski/pngquant) (lossy compressor) is used with flags:
```

Comment:
Huh. Really? PNG optimization is lossy every time? Wow.

Reply:
![image](https://user-images.githubusercontent.com/9103819/122435981-e4d11200-cfb1-11eb-8a3a-b016fb545a38.png)